app/dashboard/models.py

```diff
@@ -1223,7 +1250,7 @@ def update_slack_integration(self, token, channel, repos):
 
     @staticmethod
     def get_network():
-        return 'mainnet' 
+        return 'mainnet'
```

Comment:
should we add rinkeby back in here?

Reply:
I think something like [this](https://github.com/gitcoinco/web/pull/1283/files#diff-0d2d86c824ef942c5a3a01c83ddfe758R1226) would be good.

Reply:
@SaptakS Yeah, that was originally in `master` for a while.  I'll re-add it here.